UnityProject/Assets/Scripts/Core/Chat/EmoteActionManager.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EmoteActionManager : MonoBehaviour
+{
+    [SerializeField]
+    private List<EmoteSO> emotes;
+
+    public static bool FindEmote(string emote, EmoteActionManager instance)
+    {
+		var emoteArray = emote.Split(' ');
```

Comment:
does this mean the emote won't work if I use ``*emote`` instead of ``/me emote``?

Reply:
Tested and noticed it didn't work so I pushed a fix.